packages/python/plotly/_plotly_utils/tests/validators/test_integer_validator.py

```diff
@@ -56,6 +59,18 @@ def test_rejection_by_value(val, validator):
 def test_acceptance_min_max(val, validator_min_max):
     assert validator_min_max.validate_coerce(val) == approx(val)
 
+# With extras
+@pytest.mark.parametrize("val", ['normal', 'bold'])
```

Comment:
Thanks @marthacryan for the PR.
Could you please clarify which of these 3 cases we are testing here?
![PR_diff](https://github.com/user-attachments/assets/12e1c908-b57e-47f5-9956-a4a60069ee0f)

BTW I prefer to test all 3 cases.

Reply:
Done